subprojects/logger/gradle-logger/src/main/kotlin/com/avito/logger/GradleLoggerPlugin.kt

```diff
@@ -85,11 +86,12 @@ public class GradleLoggerPlugin : Plugin<Project> {
                 LoggerService::javaClass.name,
                 LoggerService::class.java
             ) {
+                val instantString = Instant.now().toString()
```

Comment:
Result string contains `:` symbol - `2021-11-12T14:36:34.262Z`.
In older HFS in macOs it converted to dir delimiter `/`.
I'd like to avoid such symbols just to be more confident.

Reply:
Fixed the format now is yyyy-mm-dd-hh-mm-ss